File: Alphabet symmetry.dart
List<int> solve(List<String> arr) {
  const alpha = "abcdefghijklmnopqrstuvwxyz";
  List<int> result = [];
  for(var i=0;i<arr.length;i++){
    var count = 0;
    for(var j=0;j<arr[i].length;j++){
      if(alpha.split("").indexOf(arr[i][j].toLowerCase())==j){
        count +=1;
      }
    }
    result.add(count);
  }
  return result;
}

File: Alphabet symmetry/Alphabet symmetry.py
def solve(strings : list[str]) -> list[int]:
    alpha = "abcdefghijklmnopqrstuvwxyz"
    tmp = []
    for string in strings:
        count = 0
        for i,s in enumerate(string):
                if alpha.index(s.lower()) == i:
                    count+=1
        tmp.append(count)
    return tmp
